main.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
 )
 
+// Generate the Terraform provider documentation using `tfplugindocs`:
+//go:generate go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs@v0.13.0
```

Comment:
Rather than specifying the version here and relying on go to download this as it's run the first time, can we add this to our `go.mod` and remove the version string?

That appears to be how the example terraform [provider](https://github.com/hashicorp/terraform-provider-scaffolding/blob/9468b0036786de1472763ca60bca425c0f45c237/main.go#L18) is [setup](https://github.com/hashicorp/terraform-provider-scaffolding/blob/243ba4948171e3902003f678c7c43ec3fafcdc20/go.mod#L6) and I think upgrading it via the normal go module process is more natural.

Reply:
good point!
updated the code